RELEASE-NOTES.txt

```diff
@@ -5,6 +5,7 @@
 * [*] [internal] Site creation: Adds a new screen asking the user the name of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16259]
 * [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [https://github.com/wordpress-mobile/WordPress-Android/issues/15989]
 * [*] Site creation: Adds a new screen asking the user the intent of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16306]
+* [*] Pages List: Added "Copy Link" button to context menu [https://github.com/wordpress-mobile/WordPress-Android/pull/16321]
```

Comment:
@EmilyGoose We should move this to the section for `19.8` that was just created yesterday afternoon, when the [release branch](https://github.com/wordpress-mobile/WordPress-Android/tree/release/19.7) for `19.7` has been cut.

Can you please do this change as a final update to the PR, and perhaps merge again with `trunk`? 🙇

Reply:
Updated release notes and merged upstrea